### ISSUE TITLE
type error with ts-loader, but not with tsc

### ISSUE THREAD

**Issue by trusktr** [TypeStrong/ts-loader#1246]
I'm on ts-loader 8.0.14.

### Expected Behaviour

`ts-loader` should have no type errors, just as `tsc` has no type errors.

### Actual Behaviour

`ts-loader` has a type error during Webpack build, but `tsc` does not (and I know `tsc` is the one that is correct)

### Steps to Reproduce the Problem

Not sure, I don't have a public repo with this problem yet, only a private repo.

**Comment by trusktr** [comment]
I can run `npx tsc -p tsconfig.json --outDir dist` and verify that there are no type errors, and the output looks fine.

The type error I get from `ts-loader` happens on a line where I know there should be no type error.

Any ideas what to try?

**Comment by appzuka** [comment]
@trusktr,  It is hard to give any answers without any further information.  It would be best if you could share a minimal repository that demonstrates the issue.  If you could provide at lease the error which you are seeing and your tsconfig.json &  webpack.config.js somebody may be able to help.

**Comment by BjoernHund** [comment]
@appzuka 
Today, we had a simular problem.

```
   // grid.ts
   export interface GridEntry {
      [key: string] : any;
   }
```

```
   // personGrid.ts
   import { GridEntry } from './grid';
   
   export interface PersonGridEntry extends GridEntry {
      name: string;
   }
```

```
   // page.ts
   import { PersonGridEntry } from './personGrid';
   const personEntry: PersonGridEntry = {
      id: 1,
      name: 'Test Person'
   };
```

Typescript compiler has no errors and ts-loader reports, `Property 'id' does not exist on type 'PersonGridEntry'`.

**Comment by appzuka** [comment]
@BjoernHund, if you can generate a minimal repository I'll take a look and let you know what I find.

**Comment by BjoernHund** [comment]
@appzuka 
Good morning and thank you for your quick answer.

This morning I started to create a small repository to reproduce this failure and was able to find the error on our side.

The error was in a npm module that we created. Here, an interface was imported with an incorrect relative path, which led to the error described.

Sometimes it helps to just talk about it and find the error from a different perspective.

Sorry again for that.
